sql/core/src/test/scala/org/apache/spark/sql/internal/SQLConfSuite.scala

```diff
@@ -17,13 +17,36 @@
 
 package org.apache.spark.sql.internal
 
+import org.scalatest.BeforeAndAfterAll
+
 import org.apache.spark.sql.{QueryTest, Row, SparkSession, SQLContext}
+import org.apache.spark.sql.execution.WholeStageCodegenExec
 import org.apache.spark.sql.test.{SharedSQLContext, TestSQLContext}
 
-class SQLConfSuite extends QueryTest with SharedSQLContext {
+class SQLConfSuite extends QueryTest with SharedSQLContext with BeforeAndAfterAll {
+  import testImplicits._
+
   private val testKey = "test.key.0"
   private val testVal = "test.val.0"
 
+  override def beforeAll() {
+    super.beforeAll()
+    sql("DROP TABLE IF EXISTS testData")
+    spark
+      .range(10)
+      .select('id as 'a, 'id as 'b, 'id as 'c, 'id as 'd)
+      .write
+      .saveAsTable("testData")
```

Comment:
instead of creating this table in `beforeAll`, can we create it just in the test case?